test/test-runctl-clusterctl.js

```diff
@@ -100,6 +102,14 @@ tap.test('runctl via clusterctl', function(t) {
   }, 'status worker id 6');
 
   t.doesNotThrow(function() {
+    expect('set-size CPUs');
```

Comment:
Should do a `set-size 0` before this, otherwise the test isn't valid if it happens to run on a 2 core system (quite possible in CI environments).